### PR TITLE
chore: Jest-inspired GitHub templates (thanks Jest!)

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -1,34 +1,96 @@
...
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to reproduce
+      description: Please link to a repository with a minimal reproduction and describe accurately how we can reproduce/verify the bug.
+      placeholder: |
+        Example steps (replace with your own):
+        1. Clone my repo at https://github.com/<myuser>/example
+        2. npm install
```
Comment (mxschmitt) [inline .github/ISSUE_TEMPLATE/bug.md:50]:
```suggestion
        2. npx playwright test
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Please follow these steps first:
+  - type: markdown
+    attributes:
+      value: |
+        ## Troubleshoot
+        If Playwright is not behaving the way you expect, we'd ask you to look at the [documentation](https://playwright.dev/docs/intro) and search the issue tracker for evidence supporting your expectation. Please make reasonable efforts to troubleshoot and rule out issues with your code, the configuration, or any 3rd party libraries you might be using.
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/bug.yml:13]: Do we want to suggest debugging tools here? https://playwright.dev/docs/running-tests#debugging-tests

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        ## Ask for help through appropriate channels
+        If you feel unsure about the cause of the problem, consider asking for help on for example [StackOverflow](https://stackoverflow.com/questions/ask) or our [Discord channel](https://aka.ms/playwright/discord) before posting a bug report. The issue tracker is not a help forum.
+  - type: markdown
+    attributes:
+      value: |
+        ## Make a minimal reproduction
+        To file the report, you will need a GitHub repository with a minimal (but complete) example and simple/clear steps on how to reproduce the bug.
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/bug.yml:23]: So we discourage self-contained inline snippets?

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      label: Expected behavior
+      description: A description of what you expect to happen.
+      placeholder: I expect to see X or Y
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Actual behavior
+      description: A clear and concise description of the unexpected behavior.
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/bug.yml:67]: "Please include any relevant output here, especially any error messages"?

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        ### Thank you for taking the time to suggest a new feature!
+  - type: textarea
+    id: description
+    attributes:
+      label: '🚀 Feature Request'
+      description: A clear and concise description of what the feature is.
+    validations:
+      required: true
+  - type: textarea
+    id: solution
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/feature.yml:17]: I think this and further ids are messed up.

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
+name: Feature Request 🚀
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/feature.yml:1]: Overall, I don't think having separate fields here help a lot. Something like description+example would be good enough for me.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    attributes:
+      value: |
+        > [!IMPORTANT]
+        > Bug reports without a minimal reproduction will be rejected.
+
+        ---
+  - type: input
+    id: version
+    attributes:
+      label: Version
```
Comment (dgozman) [inline .github/ISSUE_TEMPLATE/bug.yml:35]: Do we want an optional field for vscode-playwright version? Or a yes/no question "are you running tests through the extension"?

**Pull request merged**